### ISSUE TITLE
Author / affiliation updates

### ISSUE THREAD

**Issue by cdeil** [gammapy/icrc2017-gammapy-poster#4]
I'm aware of the follwing updates needed to the authors / affiliations:
- [x] Add Catherine on poster
- [x] Add Arjun on poster
- [x] Jason: change affiliation MPIK -> University of Oxford
- [x] Update affiliation on poster as here: https://github.com/gammapy/icrc2017-gammapy-poster/pull/7/files

If there's something else, please comment here!

**Comment by cdeil** [comment]
Done.

**Comment by cdeil** [comment]
@watsonjj - Apologies that your affiliation still wasn't correct.

I did change it, but then it was changed back by accident in https://github.com/gammapy/icrc2017-gammapy-poster/commit/3cbff65ab1c242a2e6509b6006097ef52cb16b29#diff-7a61539198cb50cbed42ebac4b61d690R50

Finally corrected in https://github.com/gammapy/icrc2017-gammapy-poster/commit/ebc39ea95f06d2a530d8e204a2f2ffb344842a1d , i.e. will be correct in the final published version. Unfortunately I don't think we can update the pre-ICRC version now (and anyways I didn't submit it because I'm not going there, i.e. I don't know for sure).

Same for the poster, fixed in 0e2bb41 , i.e. should be correct in the version we submit to Indico, but the printed version is already sent to the MPIK print address.
